### DISCUSSION TITLE
How can i replace an item on the ground?

### DISCUSSION THREAD

**Discussion by CaoTrongThang** [FabricMC/fabric-api#4210] [FAQ]
So i'm trying to make a mod that if players drop a raw_cod near a campfire, it'll be cooked into cooked_cod but I don't know how to get the items near the campfire and replace it by a new one. Really hope someone can help me out, thanks.

**Comment by rilakuma123** [comment]
FAQ stands for frequently asked questions. It is not Q&A (question and answer). The category you want is mod dev support.
